app-shells/mksh/mksh-57.ebuild

```diff
@@ -19,13 +29,14 @@ S="${WORKDIR}/${PN}"
 src_compile() {
 	tc-export CC
 	# we want to build static with klibc
-	if use static ; then
+	if use static
+	then
 		unset CC
 		export CC="/usr/bin/klcc"
 		export LDSTATIC="-static"
 	fi
 	export CPPFLAGS="${CPPFLAGS} -DMKSH_DEFAULT_PROFILEDIR=\\\"${EPREFIX}/etc\\\""
-	sh Build.sh -r || die
+	sh Build.sh -r -c lto || sh Rebuild.sh || die
```

Comment:
I don't think this works... what are you trying to do here? Do both files exists, or does Build.sh create the 2nd file? Anyway if this logic is correct, try with
`(sh Build.sh -r -c lto || sh Rebuild.sh) || die` instead. This only works with EAPI-7.

Reply:
It didn’t seems much wrong with a quick glance, and this is from the live ebuild.

Doing your proposed change for EAPI-7.

Reply:
In the end `Rebuild.sh` looks to have been removed from mksh, I guess I might have looked at a old cvs checkout.